common/k8stest/util_deploy.go

```diff
@@ -466,3 +466,14 @@ func RestoreDeploymentReplicas(deploymentName string, namespace string, timeout_
 	}
 	return err
 }
+
+func CheckDeploymentRunning(name, namespace string, expectedReplicas int) (bool, error) {
+	deployment, err := GetDeployment(name, namespace)
+	if err != nil {
+		return false, err
+	}
+	if deployment.Status.ReadyReplicas == int32(expectedReplicas) {
+		return true, nil
+	}
+	return false, nil
```

Comment:
also log ready and expected replica count before return

Reply:
Fixed